client.go

```diff
@@ -97,6 +97,11 @@ func (c *Client) SubscribeWithContext(ctx context.Context, stream string, handle
 		}
 		defer resp.Body.Close()
 
+		// Successful connection: reset the backoff time.
```

Comment:
This is a little problematic as reconnection may also be invoked after a read error (not just after a request error), and here at the point of reading the response, the backoff will have been reset.